src/native/routes/index.js

```diff
@@ -67,6 +66,7 @@ const Index = (
         title='Catalogos'
         icon={() => <Icon name='book' {...DefaultProps.icons} />}
         {...DefaultProps.navbarProps}
+        initial
```

Comment:
What was the reason for this move?

Reply:
ah for dev. so i didnt have to swipe each time. that should go back to noticias. good catch